tests/aiohttp/flask_app/app1.py

```diff
@@ -68,12 +69,31 @@ def testAPI1():
       response.data = str('{}')
       return response
 
+@server.route("/route1", methods=['POST'])
+def testAPI1():
+    try:
+      logger.info('Serving request for /route1')
+      loop = asyncio.get_event_loop()
+      response1 = flask.Response(mimetype='application/json')
+      response1.data = str('{ "a": "a", "xyz": "xyz" }')
+      return response1
+    except:
+      logger.error('Failed to initialize postgresql instrumentation wrapper: exception=%s, stacktrace=%s',
+        sys.exc_info()[0],
+        traceback.format_exc())
+      response = flask.Response(mimetype='application/json', status=500)
+      response.data = str('{}')
+      return response
+
 @server.teardown_request
 def after_request(response):
   try:
     logger.debug("after_request() called")
     if not ENABLE_INSTRUMENTATION:
       return response
+    if request.base_url != '/route1':
```

Comment:
Is this right? I am not exactly sure what these tests for but if we are receiving a request to a non expected endpoint we should fail I think?